packages/pwa-kit-create-app/assets/templates/@salesforce/retail-react-app/config/default.js.hbs

```diff
@@ -19,6 +19,22 @@ module.exports = {
             // This boolean value dictates whether the plus sign (+) is interpreted as space for query param string. Defaults to: false
             interpretPlusSignAsSpace: false
         },
+        login: {
+            passwordless: {
+                // Enables or disables passwordless login for the site. Defaults to: false
+                enabled: false,
+                // The callback URI, which can be an absolute URL (including third-party URIs) or a relative path set up by the developer.
+                // Required in 'callback' mode; if missing, passwordless login defaults to 'sms' mode, which requires Marketing Cloud configuration.
+                // callbackURI: 'https://webhook.site/27761b71-50c1-4097-a600-21a3b89a546c'
```

Comment:
We plan to use SendGrid OOTB and have a story for implementing it. So we'll update this value during that ticket